augur/db/__init__.py

```diff
@@ -0,0 +1,19 @@
+"""
+Commands to interface with a database.
+"""
+from augur.argparse_ import add_command_subparsers
+from augur.utils import first_line
+from . import load, export
+
+SUBCOMMANDS = [
+    load,
+    export,
+]
```

Comment:
I chose the name `load` instead of `import` because `augur import` is already a thing. I'm not familiar with it but it seems very different. It's also common to say "_load_ data into a database".

`export` may be confusing since `augur export` is an unrelated thing, but I can't think of a better term for "getting data out of a database".

Reply:
A couple disjoint thoughts:

 - I wouldn't be too confused about another `import` as a subcommand of `augur db`.
 - I prefer the parallel nomenclature of import/export over load/export.
 - As an alternative to "export", "dump" is a common term to describe getting data out of a database. Even if it has a slightly different technical meaning that how we would use it here.

Reply:
Concur with John here. load/dump and import/export are the common pairs, and I would suggest not mixing and matching them as load/export.

Reply:
Thanks for the feedback. I'll continue with import/export.

Reply:
Renamed `load` to `import` in [force-push](https://github.com/nextstrain/augur/compare/26d2f3ee405fd6c4ae0130b442bf2c1160e5ac01..07a7f047f7fbb110fb48da2f6c951d62316c0a28).